src/stores/counter.ts

```diff
@@ -0,0 +1,12 @@
+import { ref, computed } from 'vue'
+import { defineStore } from 'pinia'
+
+export const useCounterStore = defineStore('counter', () => {
+  const count = ref(0)
+  const doubleCount = computed(() => count.value * 2)
+  function increment() {
+    count.value++
+  }
+
+  return { count, doubleCount, increment }
+})
```

Comment:
I think you can remove this auto generated store since it's not used.

Reply:
In addition, since you are defining the stores inside `/core` you can remove this `/stores` folder as well.